src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/clipboard.cs

```diff
@@ -1,768 +1,497 @@
-                }
-
-                if (--i == 0)
-                {
-                    SecurityHelper.ThrowExceptionForHR(hr);
-                }
-
-                Thread.Sleep(OleRetryDelay);
-            }
-        }
-
-        /// <summary>
-        /// Get audio data as Stream from Clipboard.
-        /// </summary>
-        public static Stream GetAudioStream()
-        {
-            return GetDataInternal(DataFormats.WaveAudio) as Stream;
-        }
-
-        /// <summary>
-        /// Get data for the specified data format from Clipboard.
-        /// </summary>
-        public static object GetData(string format)
-        {
-            ArgumentNullException.ThrowIfNull(format);
-
-            if (format == string.Empty)
-            {
-                throw new ArgumentException(SR.DataObject_EmptyFormatNotAllowed);
-            }
-
-            return GetDataInternal(format);
-        }
+    public static void Clear() => ClipboardCore.Clear().ThrowOnFailure();
 
-        /// <summary>
-        /// Get the file drop list as StringCollection from Clipboard.
-        /// </summary>
-        public static StringCollection GetFileDropList()
-        {
-            StringCollection fileDropListCollection;
-            string[] fileDropList;
-
-            fileDropListCollection = new StringCollection();
-
-            fileDropList = GetDataInternal(DataFormats.FileDrop) as string[];
-            if (fileDropList != null)
-            {
-                fileDropListCollection.AddRange(fileDropList);
-            }
-
-            return fileDropListCollection;
-        }
-
-        /// <summary>
-        /// Get the image from Clipboard.
-        /// </summary>
-        public static BitmapSource GetImage()
-        {
-            return GetDataInternal(DataFormats.Bitmap) as BitmapSource;
-        }
-
-        /// <summary>
-        /// Get text from Clipboard.
-        /// </summary>
-        public static string GetText()
-        {
-            return GetText(TextDataFormat.UnicodeText);
-        }
-
-        /// <summary>
-        /// Get text from Clipboard.
-        /// </summary>
-        public static string GetText(TextDataFormat format)
-        {
-            if (!DataFormats.IsValidTextDataFormat(format))
-            {
-                throw new InvalidEnumArgumentException(nameof(format), (int)format, typeof(TextDataFormat));
-            }
+    /// <summary>
+    ///  Return <see langword="true"/> if Clipboard contains the audio data. Otherwise, return <see langword="false"/>.
+    /// </summary>
+    public static bool ContainsAudio() => ContainsDataInternal(DataFormats.WaveAudio);
 
-            string text;
+    /// <summary>
+    ///  Return <see langword="true"/> if Clipboard contains the specified data format. Otherwise, return <see langword="false"/>.
+    /// </summary>
+    public static bool ContainsData(string format) => !string.IsNullOrWhiteSpace(format) && ContainsDataInternal(format);
```

Comment:
```suggestion
    public static bool ContainsData(string format)
    {
        ArgumentException.ThrowIfNullOrEmpty(format);
        return ContainsDataInternal(format);
    }
```
Would return `False` if format is null or empty instead of throwing exception like before.